spec.html

```diff
@@ -6551,6 +6551,9 @@ <h1>SetMutableBinding ( _N_, _V_, _S_ )</h1>
           <emu-alg>
             1. Let _envRec_ be the object Environment Record for which the method was invoked.
             1. Let _bindings_ be the binding object for _envRec_.
+            1. Let _value_ be ? HasProperty(_bindings_, _N_).
+            1. If _value_ is *false*, then
+              1. If _S_ is *true*, throw a *ReferenceError* exception.
```

Comment:
I would prefer a better name than `_value_` and I think we can combine the nested ifs.

```suggestion
            1. Let _stillExists_ be ? HasProperty(_bindings_, _N_).
            1. If _stillExists_ is *false* and _S_ is *true*, throw a *ReferenceError* exception.
```

On the topic of names, the use of `S` instead of `strict` (and likewise for `N` and `V`) is unfortunate.

Reply:
The current PR matches the structure used in <https://tc39.es/ecma262/#sec-object-environment-records-getbindingvalue-n-s>, so any changes in that regard should probably happen throughout the whole clause 8.1.1. For example changing `value` to `foundBinding` to match 8.1.1.2.1.

Reply:
I don't really care about consistency in aliases, especially when we have the opportunity to make an incremental improvement. And the structure in the section you linked can't be simplified in the way I suggested here.

Reply:
> I don't really care about consistency in aliases

To be clear, when you say "aliases", are you excluding parameter names?
- I agree that `_value_` is a bad name here, and `_stillExists_` or `_foundBinding_` would be better.
- But `S`, `N`, and `V` are parameter names that appear multiple times across 8.1.1, and should probably be changed all-or-nothing, rather than incrementally.

Reply:
I would love the parameter names changed also, but since they're not being touched by this PR, I'm not going to ask for them to be changed. But I am going to ask that lines that *are* touched use better names.

Reply:
well, picking better names is a subjective game. I'm accepting the changes here but I'd still try to follow perceived convention and/or consistency from the existing parts.

Is there anything else beyond this commit suggestion I could do to unblock this PR?